.gitleaks.toml

```diff
@@ -7,6 +7,10 @@ useDefault = true # SEE: https://github.com/gitleaks/gitleaks/blob/master/config
 description = "allow ="
 Regexes = '''========================================='''
 
+[[allowlist.files]]
+pattern = '''^tests/ui/artifacts/accessibilityReport\.html$'''
```

Comment:
This file should be in gitignore not in gitleaks

Reply:
removed on [423c36c](https://github.com/NHSDigital/uec-cm/pull/80/commits/423c36c92e4e3b169d1bf4f7fa2b531700d9add6)